snap/info.go

```diff
@@ -1075,3 +1075,20 @@ func InstanceName(snapName, instanceKey string) string {
 	}
 	return snapName
 }
+
+// ByType sorts the given slice of snap info by types. The most
```

Comment:
supports sorting

Reply:
Will fix in one of the followups to avoid another spread run